src/Api/Auth/Controllers/AccountsController.cs

```diff
@@ -339,15 +339,15 @@ await _providerUserRepository.GetManyOrganizationDetailsByUserAsync(user.Id,
     }
 
     [HttpGet("organizations")]
-    public async Task<ListResponseModel<ProfileOrganizationResponseModel>> GetOrganizations()
+    public async Task<ListResponseModel<ProfileMemberOrganizationResponseModel>> GetOrganizations()
```

Comment:
Ah, sorry about the code review bingo triggered by this naming change. You can undo it if you want.